tests/serverpod_test_server/test_e2e/json_protocol_test.dart

```diff
@@ -61,7 +61,12 @@ void main() {
       WebSocketChannel websocket = WebSocketChannel.connect(
         Uri.parse(serverEndpointWebsocketUrl),
       );
-      message = await websocket.stream.asBroadcastStream().first;
+      await for (final event in websocket.stream) {
+        if (event is String && event.contains('serverOnlyScopedFieldModel')) {
+          message = event;
+          break;
+        }
+      }
```

Comment:
Question: What happens if the message is never sent? This will loop forever right?

Reply:
Same concern for the next test.

Reply:
Fixed!